Projektdoku/Gespraeche.tex

```diff
@@ -0,0 +1,257 @@
+\item Absprache welche Tools für die interne Gruppenorganisationen genutzt werden. 
+\end{itemize}
+
+\subsubsection{Rollen}
+\begin{itemize}
+	\item Projektleiter: Leo Lindhorst
+	\item Analyse : Oliver von Seydlitz
+	\item Entwurf (Grob-, Fein-): Denis Keiling
+	\item Implementierung: Leo Lindhorst
+	\item Test: Oliver von Seydlitz
+	\item Datenbank: Hung Nguyen
+	\item Dokumentation: Fabian Krehnke 
+	\item Qualitätssicherung: Denis Keiling 
+\end{itemize}
+
+\subsubsection{Tools}
+\begin{itemize}
+	\item GitHub und Git für Versionsverwaltung 
+	\item Kommunikation: Slack und Email
+	\item Projektmanagement: Clickup
+\end{itemize}
+
+
+
+\subsection{Erstes Meilensteintreffen 02.05.} 
+
+Zeit: 13.20 - 14.00
+
+\subsubsection{Anwesenheit}
+\begin{itemize}
+	\item Herr Zirkelbach, Ansprechpartner in diesem Termin war Herr Zirkelbach da Frau Hauptmann Krankheitsbedingt den Termin nicht wahrnehmen konnte.
+	\item Leo Lindhorst
+	\item Oliver von Seydlitz
+	\item Denis Keiling
+	\item Hung Nguyen
+	\item Fabian Krehnke
+\end{itemize}
+
+\subsubsection{Inhalt}
+\begin{itemize}
+\item Es wurden Fragen der Gruppenmitglieder über die zugeteilten    Themengebiete geklärt.
+\item Vorstellung der von Oliver von Seydlitz erstellten Anforderungsanalyse für das Pflichtenheft. 
+\item Besprechung der Termine bzw Zeiträume die in dem Projekt vorhanden sein sollen.
+\end{itemize}
+
+\subsubsection{Ergebnis}
+Die gesperrten Termine werden über den Zustand "deaktiviert" abgebildet. Es ist immer ein Termin als Ganzes (d.h. ein Tag) aktiviert oder deaktiviert.
+
+
+\subsection{Kurzgespräch 09.05.}
+
+Zeit:12:40 - 12:50 
+
+\subsubsection{Anwesenheit}
+\begin{itemize}
+	\item Frau Hauptmann
+	\item Leo Lindhorst
+	\item Oliver von Seydlitz
+	\item Denis Keiling
+	\item Hung Nguyen
+\end{itemize}
+
+\subsubsection{Inhalt}
+\begin{itemize}
+\item Besprechung mit Frau Hauptmann über die möglichen Zustände innerhalb der Anwendung. Der Zustand \textit{gebucht nicht bestätigt} wurde auf Wunsch von Frau Hauptmann entfernt. 
+\item Vorschlag von Frau Hauptmann das Projekt als Web- oder Desktop-Anwendung umzusetzen.
+\item Team-intern wurde mittels einer Umfrage entschieden, das Projekt als Desktop-Anwendung zu realisieren.
+\end{itemize}
+
+\subsubsection{Termine}
+Den 16.5. als Termin für das 2. Meilensteintreffen vereinbart.
+
+
+
+\subsection{Zweites Meilensteintreffen 16.05.}
+
+Zeit: 14.15 - 15.15
+
+\subsubsection{Anwesenheit}
+\begin{itemize}
+	\item Frau Hauptmann
+	\item Leo Lindhorst
+	\item Oliver von Seydlitz
+	\item Denis Keiling
+	\item Hung Nguyen
+	\item Fabian Krehnke
+\end{itemize}
+
+\subsubsection{Inhalte}
+\begin{itemize}
+\item Vorstellung des Pflichtenheftes durch Oliver und der UI-Prototypen von Leo.
+\item Besprechung der nächsten Schritte, insbesondere Grob- u. Feinentwurf.
+\end{itemize}
+
+\subsubsection{Feedback}
+
+\paragraph{Pflichtenheft}
+\begin{itemize}
+\item Das Pflichtenheft wurde in Markdown erstellt und die enthaltenen
```

Comment:
Ich verstehe nicht, warum das unter Feedback steht

Reply:
bei Inhalte passt es ja auch nicht, hättest du einen vorschlag?

Reply:
Ich schlage vor, all diese Art Anmerkungen kommen in eine extra Section in der Entwurfsdoku

Reply:
Könnte dann "verwendete Technologien" heißen oder so. Da steht dann drin, womit wir die Diagramme gemacht haben, den Projektablaufplan, die Oberflächenprototypen usw

Reply:
klingt gut